src/Cassandra/Connections/Connection.cs

```diff
@@ -797,6 +797,7 @@ private void RunWriteQueueAction()
                 }
                 _pendingOperations.AddOrUpdate(streamId, state, (k, oldValue) => state);
                 int frameLength;
+                var startPosition = stream?.Position ?? 0; 
```

Comment:
Reading through `RecyclableMemoryStream.cs` it looks like it makes more sense to replace this with

`var startLength = stream?.Length ?? 0;`

@jorgebay do you agree?

Reply:
In this case is the same, I'm ok with `Position` (equivalent to [writer index](https://netty.io/4.1/api/io/netty/buffer/ByteBuf.html#writerIndex--) in Netty).

Reply:
hm... now that I think about it, as below we are setting the `Length` (that sets `Position` internally) it makes sense to use 1 property: `Length`.

Reply:
:+1: @collinsauve can you change `stream?.Position` to `stream?.Length` and rename the `startPosition` variable accordingly? After that change and CI run is done I'll merge this.

Reply:
done